src/Tools/ExternalAccess/Razor/Microsoft.CodeAnalysis.ExternalAccess.Razor.csproj

```diff
@@ -20,6 +20,7 @@
     <!--
       ⚠ ONLY RAZOR ASSEMBLIES MAY BE ADDED HERE ⚠
     -->
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.Razor.LanguageServer" Key="$(RazorKey)" />
```

Comment:
We're planning on using it in `Microsoft.AspNetCore.Razor.LanguageServer.CodeActions`, I don't believe the last part needs to be included, correct?